gothemis/session/session_test.go

```diff
@@ -223,6 +223,29 @@ func testSession(keytype int, t *testing.T) {
 	<-finCh
 }
 
+func testInvalidKeyHandling(t *testing.T) {
+	kpa, err := keys.New(keys.TypeRSA)
+	if nil != err {
+		t.Error(err)
+		return
+	}
```

Comment:
Within Go conditions assignments are not allowed. Therefore, a customary form of error checks exists:

```go
if err != nil {
        // handle it
}
```

Also, `testing.T` has an alternate method for reporting fatal errors:

```go
t.Fatal("sum ting wong", err)
```

which immediately stops the test after reporting an issue.

Reply:
> Are more wrappers expected in this PR of this is all for now?

No, that's all. But I would appreciate if you can add that little check you mentioned earlier to ObjC wrapper and test it on some Apple device (at least, compilation). And then maybe show `git format-patch` result so I can do `git am`. That shouldn't affect existing ObjC tests as well as all the code that actually uses this wrapper, I hope.